templates/ammos-cubs-ait.template.yaml

```diff
@@ -73,6 +95,78 @@ Mappings:
       RHELHA84HVM: ami-0cca63ccd9a87f1b2
 
 Resources:
+  EfsSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      VpcId:
+        Ref: VPCID
+      GroupDescription: "EFS Security Group"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 2049
+          ToPort: 2049
+          CidrIp: 0.0.0.0/0
+
+  FileSystem:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      AvailabilityZoneName: !Ref 'AvailabilityZone'    
+      Encrypted: true
+      PerformanceMode: generalPurpose
+      BackupPolicy:
+        Status: DISABLED
+      FileSystemTags:
+      - Key: Name
+        Value: ait-efs
+
+  MountTargetPublicA:
+    Type: AWS::EFS::MountTarget
+    Properties:
+      FileSystemId: !Ref FileSystem
+      SubnetId: "subnet-03a1ec6be37d72106"
```

Comment:
🙅 no hard-coding

Reply:
also, we need to make sure that one MountTarget is created for each SN that is used by the ASG

Reply:
how come we need one for each?